packages/wdio-cli/src/utils.ts

```diff
@@ -135,21 +147,21 @@ export function findInConfig(config, type) {
     return config.match(regex)
 }
 
-export function replaceConfig(config, type, name) {
+export function replaceConfig (config: string, type: string, name: string) {
+    if (type === 'framework') {
```

Comment:
is it ok that the condition order changed?

Reply:
that should be irrelevant